tests/model/generated/object/test_notification_url.py

```diff
@@ -65,8 +67,12 @@ class TestNotificationUrl(bunq_test.BunqSdkTestCase):
     def execute_notification_url_test(self,
                                       file_path,
                                       class_name,
-                                      getter_name):
+                                      getter_name,
+                                      sub_class_expected_object_name=None,
```

Comment:
Spaces around `=`?

Reply:
This is python standard for optional arguments . :thumbsup: